### ISSUE TITLE
docs clarification - more details about activating virtual environment? `make requirements` fails?

### ISSUE THREAD

**Issue by danbjoseph** [AmericanRedCross/street-view-green-view#19]
on an Apple M2 Pro, macOS 13.4.1 (c) 

after `make create_environment` the log reads
```
>>> New virtualenv created. Activate with:
workon street-view-green-view
```
but that command errors with `command not found: workon`
progressing to step 2 failed until I ran `source venv/bin/activate` which is not mentioned in the docs
even after activating the virtual environment, running `make requirements` failed with
```
/opt/homebrew/Cellar/python@3.11/3.11.5/Frameworks/Python.framework/Versions/3.11/Resources/Python.app/Contents/MacOS/
Python: can't open file '/Users/danbjoseph/GitHub/americanredcross/street-view-green-view/test_environment.py': 
[Errno 2] No such file or directory
make: *** [test_environment] Error 2
```
however, running `pip install .` did work

<img width="971" alt="Screenshot 2024-03-02 at 10 47 28 AM" src="https://github.com/AmericanRedCross/street-view-green-view/assets/4806884/3db2c5a4-e58c-4e72-bc31-dd93339258ef">

**Comment by dragonejt** [comment]
To be honest, I didn't use the makefile either and I don't know what test_environment.py is, it's not something I've added or changed. I can take a look at updating the Makefile to how I am currently running it.

**Comment by danbjoseph** [comment]
@jayqi was this something you worked on?

**Comment by jayqi** [comment]
@danbjoseph there were two things going on here that I think should be addressed by #21
